recipes/leveldb/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+
+### Build and check
+
+make
+make check
+
+
+### Install manually ( no make install :) )
```

Comment:
As noted: Unfortunately, then didn't add a `make install` command. I expect this is partially because they don't run configure so they don't determine what prefix to install to. Fortunately, it isn't too bad to copy things over.

I manually inspected what got built by forcing a failure at the end of the `make check` step and this is what I found to be useful. The rest is `*.o` files, `*.cc` files, and `*.h` files.